### PR TITLE
refactor(kk theme): load fonts from google/typography.com

### PR DESCRIPTION
**Pull request by evoX0**
Currently, KK theme object is empty due to heavyweight.

Load open sans from google and the didot font from typography.com instead of hard coding. Corresponds to the fix in dbmedialab/kk-styleguide#67

### CODE REVIEW

**Comment by misund** [comment]
If we are to switch to Google Fonts, this PR is fine and dandy. However, the Didot from Google Fonts sadly is not the same font as the Didot from font-specs. Also, we should probably match the fonts in https://github.com/dbmedialab/kk-styleguide/blob/bc232140a80d0428efe77325a5911e3f7a9f9242/scss/base/_fonts.scss

A switch to the Google Fonts version would be lighter on the page, but we'd need designer approval. @aprestmo is the last one to change the fonts in the kk-styleguide. Would you want me to establish contact?

File: src/themes/kk/index.js [hunk 1]
```diff
@@ -13,7 +12,8 @@ const variables = {
 };
 
 const global = `
-	${fontSpecs}
+	@import url('https://fonts.googleapis.com/css?family=GFS+Didot');
```
[the remark this answers is not in the document]
Comment (misund) [reply src/themes/kk/index.js]: English next time, pls

[the remark this answers is not in the document]
Comment (dlazarchuk) [reply src/themes/kk/index.js]: > English next time, pls

fixed, sorry)

**Comment by evoX0** [comment]
> A switch to the Google Fonts version would be lighter (kB-wise), but we'd need designer approval, because the fonts are not the same. @aprestmo is the last one to change the fonts in the kk-styleguide. Would you want me to establish contact?

Yep, your right, we need design approval.
Sadly we can't add this woff to GF library, or I'm wrong?
That will be great if you chat with @aprestmo. Thanks in advance

**Comment by misund** [comment]
Also, please include a description of the changes in the PR title

**Comment by evoX0** [comment]
> Spoke with @aprestmo. The fonts are not the same, so we can not switch to GFS Didot.
> 
> However, it turns out that the fonts are doubly defined in the KK styleguide. In addition to the base64-encoded fonts that we have copied to shiny, the fonts are also imported from https://cloud.typography.com/7308714/6266192/css/fonts.css
> 
> We should clean this up by removing the base64-encoded fonts from both kk-styleguide and shiny, and to import this file in shiny.
> 
> @aprestmo will fix this in kk-styleguide. Will you fix it in shiny, @evoX0 ?

As far as I understood, we just store this woff's localy without encoding?

**Comment by misund** [comment]
Not sure if I was clear. @evoX0, can we do
```css
@import url('https://cloud.typography.com/7308714/6266192/css/fonts.css');
```

instead of providing the fonts ourselves?

**Pull request merged**